MiniAn/minian_main.py

```diff
@@ -435,7 +438,113 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(minian_parameters, currentFile_AC, currentFile_A):
+    performCrossReg = minian_parameters.params_crossRegister["crossReg"]
+    if not performCrossReg:
+        return currentFile_A
+    param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
+    refFileName = minian_parameters.params_crossRegister["fname"]
+    refImages   = minian_parameters.params_crossRegister["h5path_images"]
+    refRois     = minian_parameters.params_crossRegister["h5path_roi"]
+
+    # Load base/reference file to xarray
+    refCR, fileCR_ =  load_doric_to_xarray(refFileName, refImages, np.float64, None, "subset", None)
+
+    # max project of the miniAn images for base file and current file
+    ref_max     = refCR.max("frame")
+    current_max = currentFile_AC.max("frame")
+
+    result = xr.concat([ref_max, current_max], pd.Index(['session1', 'session2'], name = "session"))
```

Comment:
try to avoid using general names, such as "result", "temps", they don't bring clarity to the code